config/deploy.rb

```diff
@@ -154,6 +156,7 @@
           execute :systemctl, '--user', :restart, :gather_vote_account_details
           execute :systemctl, '--user', :restart, :process_ping_thing
           execute :systemctl, '--user', :restart, :front_stats_update
+          execute :systemctl, '--user', :restart, :leader_stats_update
```

Comment:
Please comment this line out. This will break the deploy. It must be done in the next step, after the script is already on the server.

Reply:
Thanks for warning me. I've commented this for restart block.